pkg/storage/store.go

```diff
@@ -149,6 +150,65 @@ func StartTask(id int) {
 	}
 }
 
+func EditTask(id int) {
+	tasks := loadTasks()
+
+	var task *Task
+
+	var index int
+	for i, t := range tasks.Tasks {
+		if t.Id == id {
+			index = i
+			task = &t
+			break
+		}
+	}
+
+	if task == nil {
+		panic("Task " + string(id) + " does not exist.")
+	}
+
+	f, err := ioutil.TempFile(os.TempDir(), "clerk")
+	handleError(err)
+
+	// Create a temporary file to allow the user to edit the task
+	_, err = f.Write([]byte(fmt.Sprintf("%s\n%s", task.Title, task.Description)))
+	handleError(err)
+
+	fpath := f.Name()
+
+	f.Close()
+
+	// Option the default editor on the OS
+	cmd := exec.Command(os.Getenv("EDITOR"), fpath)
```

Comment:
How does this know what file to edit?

Reply:
`fpath` is the path to the temporary file I created, which is passed in as an argument.

Reply:
Ah, so the file being closed doesn't affect the call to the editor, makes sense. Thanks

Reply:
No. I'm not sure if we need to close the file or not, but vim is taking over.